src/video/emscripten/SDL_emscriptenvideo.c

```diff
@@ -218,14 +218,19 @@ Emscripten_CreateWindow(_THIS, SDL_Window * window)
     SDL_WindowData *wdata;
     double scaled_w, scaled_h;
     double css_w, css_h;
+    const char *canvasElement = SDL_GetHint(SDL_HINT_EMSCRIPTEN_CANVAS_ELEMENT);;
```

Comment:
Should be `canvas_element` (or even `canvas_selector`), also there's an extra semicolon.

Reply:
ok for the semicolon, I've updated the PR, but I've used `canvasElement` according to:
https://github.com/libsdl-org/SDL/blob/main/src/video/emscripten/SDL_emscriptenevents.c#L722

and anyway all the code calls it element, e.g. `emscripten_set_canvas_element_size`
The variable name is not really a problem for me, I can change it

Reply:
Yeah, that one is also wrong... most names are in snake_case now. (You could argue that that one is even more wrong as some of the options aren't even elements... But uh, it is also 8 year old code :smile: )

... and yeah there's a bit of inconsistent naming going on, mostly because it used to be an element id(+ some magic values) and is now a CSS selector (+ some other magic values).

Reply:
so? @slouken @Daft-Freak going for `canvas_element` or `canvas_selector`?

Reply:
and yes I handle directly `specialHTMLTargets` object because my canvas is in a shadow dom, emscripten wouldn't be able to find it without the shadow dom root